tutorials/cookbook/Recipes Thomas.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>Maybe add one line description of what we're trying to achieve with that recipe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>